apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/class-wp-rest-wpcom-block-editor-nux-status-controller.php

```diff
@@ -79,7 +79,15 @@ public function get_nux_status() {
 			$variant = 'tour';
 		}
 
-		if ( has_filter( 'wpcom_block_editor_nux_get_status' ) ) {
+		$blog_id   = get_current_blog_id();
+		$is_p2     = \WPForTeams\is_wpforteams_site( $blog_id );
```

Comment:
WPForTeams might be available globally but probablygood to require in case? Does this work on Atomic?

Reply:
This is part of public-api. I don't think Atomic vs Simple matters.

Reply:
Turns out you're right. This doesn't work on Atomic.